src/routes/(console)/organization-[organization]/billing/planSummary.svelte

```diff
@@ -59,147 +60,160 @@
                     $organization?.billingNextInvoiceDate
                 )}
             </p>
-            <Collapsible>
-                <CollapsibleItem noContent>
-                    <span class="body-text-2">
-                        {tierToPlan($organization?.billingPlan)?.name} plan</span>
-                    <div class="body-text-2 u-margin-inline-start-auto">
-                        {isTrial
-                            ? formatCurrency(0)
-                            : currentPlan
-                              ? formatCurrency(currentPlan?.price)
-                              : ''}
-                    </div>
-                </CollapsibleItem>
-                {#if $organization?.billingPlan !== BillingPlan.FREE && extraUsage > 0}
-                    <CollapsibleItem isInfo gap={8}>
-                        <svelte:fragment slot="beforetitle">
-                            <span class="body-text-2"><b>Add-ons</b></span><span class="inline-tag"
-                                >{extraMembers ? extraAddons + 1 : extraAddons}</span>
-                            <div class="icon">
-                                <span class="icon-cheveron-down" aria-hidden="true"></span>
-                            </div>
-                        </svelte:fragment>
-                        <svelte:fragment slot="end">
-                            <div class="body-text-2 u-margin-inline-start-auto">
-                                <b>
-                                    {formatCurrency(extraUsage >= 0 ? extraUsage : 0)}
-                                </b>
-                            </div>
-                        </svelte:fragment>
+            {#if !isLoading}
```

Comment:
nit: for readability, it might be good to put the loading condition first because the block is smaller and you can understand the if/else easier. With the else block down below, i may forget what the if condition is by the time I get down there.